docs/Reference/ID-Prefix-Reservation.md

```diff
@@ -0,0 +1,156 @@
+---
+# required metadata
+
+title: ID Prefix Reservation Reference | Microsoft Docs
+author: dajaco
+ms.author: dajaco
+manager: unnir
+ms.date: 10/9/2017
+ms.topic: reference
+ms.prod: nuget
+#ms.service:
+ms.technology: null
+#ms.assetid: 
+
+# optional metadata
+
+description: Package ID Prefix Reservation feature description and author guide.
+keywords: NuGet package ID, prefix, reservation
+#ROBOTS:
+#audience:
+#ms.devlang:
+ms.reviewer:
+- anangaur
+- karann
+- unnir
+#ms.suite:
+#ms.tgt_pltfrm:
+#ms.custom:
+
+---
+
+# Package ID prefix reservation
+[NuGet.org](https://www.nuget.org/) and Visual Studio 2017 version 15.4 or later now show a visual indicator for packages that are submitted by owners with a reserved package ID prefix, as long as the package matches the reserved ID prefix naming pattern. The below reference explains what the ID prefix reservation entails, and how an owner can apply for an ID prefix.
+
+## What is package ID prefix reservation?
+We've heard from the NuGet community that there are identity issues when trying to consume NuGet packages. We've also heard from NuGet package owners that sometimes they are forced to make takedown requests for packages that are squatting on their ID prefixes - something that can cause delays in shipping new packages. To alleviate pain in both the package consumer and package owner scenarios, we have built a package ID prefix reservation feature.
```

Comment:
This sounds like a blog post, I don't thing we need to explain the reason why we are implementing this feature. Sure, we can call out the advantages,

Reply:
Updated